src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int lengthFirstBase;
+    private int lengthSecondtBase;

```

Comment:
There's a typo in the variable name 'lengthSecondtBase'. It should be 'lengthSecondBase'. Remember to update it in the constructor and other methods as well.